Source/API/EbApi.h

```diff
@@ -213,6 +223,13 @@ typedef struct EB_H265_ENC_CONFIGURATION
     // Debug tools
     uint32_t                reconEnabled;
 
+    // SEI
+    uint16_t                maxCLL;
+    uint16_t                maxFALL;
+    const char*             masteringDisplayColorVolume;  // SMPTE ST 2086 mastering display color volume info
+    uint32_t                dolbyVisionProfile;
+    const char*             naluFile;
```

Comment:
These pointers should not be passed into the library, especially if they are file pointers. This will increase the vulnerability of our encoder.